mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -105,4 +105,7 @@ SearchException cannotFindReadableProperty(@FormatWith(PojoTypeModelFormatter.cl
 
 	@Message(id = ID_OFFSET_2 + 11, value = "Mapping service cannot create a search manager using a different session factory. Expected: '%1$s'. In use: '%2$s'.")
 	SearchException usingDifferentSessionFactories(SessionFactory expectedSessionFactory, SessionFactory usedSessionFactory);
+
+	@Message(id = ID_OFFSET_2 + 12, value = "Exception while retrieving property type model for '%1$s' on '%2$s'")
+	SearchException errorRetrivingPropertyTypeModel(String propertyModelName, String parentTypeModel, @Cause Exception cause);
```

Comment:
`errorRetrivingPropertyTypeModel` => `errorRetrivingPropertyTypeModel`

Reply:
Also, you could pass the type model directly instead of using toString(). Just use a formatter, like on `org.hibernate.search.mapper.pojo.logging.impl.Log#unableToResolveDefaultIdentifierBridgeFromSourceType` for example.